SCons/Taskmaster/__init__.py

```diff
@@ -159,7 +168,7 @@
         """
         global print_prepare
         T = self.tm.trace
-        if T: T.write(self.trace_message('Task.prepare()', self.node))
+        if T: self.trace_message(self.node)
```

Comment:
> __[Flake8]__ multiple statements on one line (colon) ([view](https://sider.review/gh/repos/104670160/pulls/4249#issue-8187616))
>
> <details>
>
> | Rule |
> | ---- |
> | `E701` |
>
> You can [close this issue](https://sider.review/gh/repos/104670160/pulls/4249#issue-8187616) if no need to fix it. [Learn more](https://help.sider.review/getting-started/working-with-issues).
>
> </details>

In the particular case of trace/debug messages, I'm happy with making them onliners. Propose we accept-and-close these.

Reply:
... although, if we ever do black the codebase, it's going to break those apart anyway....

Reply:
Fixed all these.